server/channels/app/plugin_install.go

```diff
@@ -383,7 +382,7 @@ func (ch *Channels) installExtractedPlugin(manifest *model.Manifest, fromPluginD
 	}
 
 	pluginPath := filepath.Join(*ch.cfgSvc.Config().PluginSettings.Directory, manifest.Id)
-	err = utils.CopyDir(fromPluginDir, pluginPath)
+	err = model.CopyDir(fromPluginDir, pluginPath)
```

Comment:
Nit: `CopyDir` doesn't fit in `model` IMO.  Do we need a public `utils` as well?  Otherwise `model` will be a dumping ground for non-model stuff.